src/senaite/core/extender/label.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from archetypes.schemaextender.interfaces import IBrowserLayerAwareExtender
+from archetypes.schemaextender.interfaces import IOrderableSchemaExtender
+from archetypes.schemaextender.interfaces import ISchemaExtender
+from archetypes.schemaextender.interfaces import ISchemaModifier
+from bika.lims import senaiteMessageFactory as _
+from Products.CMFCore import permissions
+from senaite.core.browser.widgets.queryselect import QuerySelectWidget
+from senaite.core.catalog import SETUP_CATALOG
+from senaite.core.config.fields import AT_LABEL_FIELD
+from senaite.core.extender import ExtLabelField
+from senaite.core.interfaces import ICanHaveLabels
+from senaite.core.interfaces import ISenaiteCore
+from zope.component import adapts
+from zope.interface import implements
+
+
+class LabelSchemaExtender(object):
+    """Extend schema fields for labeled contents
+    """
+    layer = ISenaiteCore
+    implements(
```

Comment:
As far as I am concerned, the decorators `@implementer` and `@adapter` are preferred nowadays.

Reply:
Yes, that's true. But for multiple implemented Interfaces I prefer this notation